samples/matter/common/src/Kconfig

```diff
@@ -80,3 +80,9 @@ config NCS_SAMPLE_MATTER_TEST_EVENT_TRIGGERS_MAX_TRIGGERS_DELEGATES
 	help
 	  Defines the maximum number for the TestEventTriggerDelegate implementations
 	  to be registered in the nRF test event triggers class.
+
+config NCS_SAMPLE_MATTER_PERSISTENT_STORAGE
+	bool "Persistent storage support for Matter samples"
+	default n
```

Comment:
I believe `default n` is redundant here.

Reply:
+1, I think it was done to increase the readability, but tbh I'm not a big fan of adding it as well (unless it's some special case that need to be clearly explained, but imo usually it just makes config files to be unnecessarily larger)

Reply:
Sure, I will remove this line.